METADATA_SUPPORT.md

```diff
@@ -374,6 +374,7 @@ To contribute a new metadata type, please see the [Contributing Metadata Types t
 |PathAssistantSettings|✅||
 |PaymentGatewayProvider|✅||
 |PaymentsManagementEnabledSettings|✅||
+|PaymentsSettings|✅||
```

Comment:
Running `yarn update-supported-metadata DigitalExperienceBundle DigitalExperience DigitalExperienceConfig` made these changes too. I wasn't sure if I should remove them or just leave them